tests/integration/suite/test_node.py

```diff
@@ -160,13 +160,22 @@ def _node_template():
     node_template = wait_for(_node_template,
                              fail_handler=lambda:
                              'failed to create node template')
-    wait_remove_resource(node_template)
 
     node_pool = client.create_node_pool(
         nodeTemplateId=node_template.id,
         hostnamePrefix="test1",
         clusterId="local")
+
+    def node_available():
+        node = client.list_node(nodePoolId=node_pool.id)
+        if len(node.data):
+            return node.data[0]
+        return None
+
+    node = wait_for(node_available)
+    wait_for_condition("Saved", "False", client, node)
```

Comment:
Why false ?

Reply:
The node that is created with a Digital Ocean driver, but the SSH information is fake and will cause the "save" to fail even though the files are created.